src/NServiceBus.Monitoring/ProcessingTime/ReportProcessingTime.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace NServiceBus.Monitoring.ProcessingTime
+{
+    using System;
+    using System.Diagnostics;
+
+    class ReportProcessingTime
+    {
+        public void Report(DateTime timeSent, string processedMessageType,  DateTime startedAt, DateTime completedAt)
+        {
+            var computedTimeElapsed = ProcessingTimeCalculator.Calculate(startedAt, completedAt);
+
+            var totalTimeInSeconds = Convert.ToInt32(computedTimeElapsed);
```

Comment:
@mikeminutillo - I think, milliseconds make more sense. Especially if there are tools that try to graph it, etc and need a more granular measurement. We can always round to seconds, but we can't get the original value back.